### PR TITLE
Migrating code to show bike parking spaces as a vector layer

### LINKED ISSUE #370: Convert bike parking layer from GeoJSON to actual vector tiles
Currently DT next uses a rarely updated GeoJSON as the data source for the bike park layer.

Screenshot from 2021-04-20 17-52-31

In DT1 we converted to an actual vector tile layer which is automatically updated every night.

PR: https://github.com/stadtnavi/digitransit-ui/pull/340

Please convert the current bike park layer in next to a vector tile one.

### PR DESCRIPTION
**Pull request by jpidelatorre**
## Proposed Changes

  -

## Review

  - Read and verify the code changes
  - Test the functionality by running the UI locally with all popular browsers available in your platform

closes #370

### CODE REVIEW

**Comment by leonardehrenfried** [comment]
This is a work in progress but here are my requests:

- Resolve merge conflicts
- Use the compilation script to add the icons: https://github.com/stadtnavi/digitransit-ui/blob/next/docs/Icons.md
- Make the black dots at high zoom level also blue
- Add `SelectBikeParkRow`: https://github.com/stadtnavi/digitransit-ui/blob/master/app/component/map/tile-layer/SelectBikeParkRow.js

File: static/assets/svg-sprite.hb-source.svg [hunk 1]
```diff
@@ -940,7 +940,7 @@
     </symbol>
     <!-- carsharing && taxi quick fix END-->
     <!--new regiorad icons START-->
-    <symbol id="icon-icon_citybike-withoutBox" viewBox="0 0 283.46 283.46">
+    <symbol id="icon-icon_city`withoutBox" viewBox="0 0 283.46 283.46">
```
Comment (leonardehrenfried) [inline static/assets/svg-sprite.hb-source.svg:910]: I think this was an accident, isn't it?

**Comment by leonardehrenfried** [comment]
@jpidelatorre Are you blocked here?

**Comment by jpidelatorre** [comment]
> @jpidelatorre Are you blocked here?

Sorry, Leonard, I didn't read your comment. I was not blocked, I was solving some merging conflicts.
I think the latest code is working properly.

**Comment by leonardehrenfried** [comment]
Great, checking it.

File: app/component/map/tile-layer/VectorTileLayerContainer.js [hunk 2]
```diff
@@ -20,6 +21,15 @@ export default function VectorTileLayerContainer(props, { config }) {
         layersToAdd.push(CityBikes);
       }
 
+      if (
+        config.dynamicParkingLots &&
+        config.dynamicParkingLots.showDynamicParkingLots
+      ) {
+        layers.push(DynamicParkingLots);
+      }
+      if (config.bikeParks && config.bikeParks.show) {
```
Comment (leonardehrenfried) [inline app/component/map/tile-layer/VectorTileLayerContainer.js:30]: You need to change this code to take the new variable name `layersToAdd` into account. Also, please remove the code about dynamic parking lots from this PR.

**Review by leonardehrenfried — CHANGES_REQUESTED** [review]
The latest version of this PR doesn't show any bike parking facilities on the map. Please read my comment.

**Comment by leonardehrenfried** [comment]
Thanks

**Pull request merged**